javascript/event-recommender.md

```diff
@@ -0,0 +1,80 @@
+# Event Recommender System with Javascript
+
+
+### Primary Goals
+To build a modular system using Object Oriented Programming principles.
+
+### Overview
+In this project, you'll build an EventRecommender System that will
+help you practice Object Oriented Programming using Javascript. This should take
+a maximum of 3 hours.
+
+### Context
+* Object Oriented Programming (OOP) is a popular style of programming that is oftentimes
+adopted for Backend Code Organization.
+* OOP enables
+    * Readability (esp in a large code base)
+    * Function abstraction
+    * Easier testing
+* Displaying Object Oriented Programming Skills in an interview is extremely impressive, and shows your ability to design and organize systems.
+
+
+### How to get Started
+#### User Requirements
+When you instantiate an EventRecommender object, you should be able to do the following:
+
+* Users:
+    * Add a new User to the System
+    * Allow users to save events to a personal Events array.
+    * Delete a User from the System
+* Event:
+    * Add a new Event to the System
+    * Delete an Event from the System
+* Filter:
+    * Note: You can use three separate filter functions or _one_ if you'd like an extra challenge :)
```

Comment:
Thanks for removing categories! Now it should say “ You can use two separate ...”